sharding/networksharding/peerShardMapper.go

```diff
@@ -191,9 +199,27 @@ func (psm *PeerShardMapper) getPeerInfoSearchingPidInFallbackCache(pid core.Peer
 		}
 	}
 
+	subTypeObj, ok := psm.peerIdSubType.Get([]byte(pid))
```

Comment:
maybe update the existing tests in peerShardMapper_test to also check the subPeerType field?

Reply:
Done